draft/index.html

```diff
@@ -847,7 +847,7 @@ <h3>Data Extension Property Proposals</h3>
             <dt><code>type</code></dt>
             <dd>The <a>type</a> identifier supplied in the query;</dd>
             <dt><code>limit</code></dt>
-            <dd>Optionally, the requested limit;</dd>
+            <dd>Optionally, an integer to control the number of proposed properties returned for the given type;</dd>
```

Comment:
Since this is the response, not the request, I'd suggest something like this:

```suggestion
            <dd>Optionally, an integer to indicate the requested limit;</dd>
```